File: backend/semgrep_rules/tests/python/kisa_sw_20_insecure_permissions.py
import os
def unsafe(path):
    # ruleid: kisa.sw20.python.world-writable-permission
    os.chmod(path, 0o777)
def safe(path):
    # ok: kisa.sw20.python.world-writable-permission
    os.chmod(path, 0o600)

File: backend/semgrep_rules/tests/javascript/kisa_sw_20_insecure_permissions.js
const fs = require("fs");
function unsafe(path) {
  // ruleid: kisa.sw20.javascript.world-writable-permission
  fs.chmodSync(path, 0o777);
}
function safe(path) {
  // ok: kisa.sw20.javascript.world-writable-permission
  fs.chmodSync(path, 0o600);
}
